internal/identity/identitymanager.go

```diff
@@ -109,6 +110,13 @@ func (im *identityManager) GetLocalNode(ctx context.Context) (node *core.Identit
 	return node, err
 }
 
+func (im *identityManager) GetRootOrg(ctx context.Context) (org *core.Identity, err error) {
+	orgName := im.multiparty.RootOrg().Name
```

Comment:
While I tracked the code through to check `RootOrg()` can't return nil, it feels a bit odd to continue this check in the case that `Name` is `""`